http/README.md

```diff
@@ -68,5 +68,84 @@ Table below shows the behaviour of HttpAction depending on provided `responseOpt
 | application/text | false     | JSON           | JSON | _error     | -        |
 | application/text | true      | JSON           | JSON | _error     | -        |
 
+### Node log
+`HttpAction` provides node log, every HTTP call is logged on `INFO` level, failing ones also on `ERROR`. 
+Every HTTP call which have ended with 200 status, but causing exception during processing in `HttpAction` 
+will be treated as succeeding, and the transition will be set to `_error`. Every error is logged.
+
+Node log is presented as `JSON`. 
+
+`JSON` structure of succeeding request on `INFO` level looks like the following (on `ERROR`
+level no logs will be produced):
+```json
+{
+  "request": {
+    "path": "/api/endpoint",
+    "requestHeaders": {
+      "Content-Type": "application/json"
+    }
+  },
+  "response": {
+    "httpVersion": "HTTP_1_1",
+    "statusCode": "200",
+    "statusMessage": "OK",
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "trailers": {
+    },
+    "httpMethod": "GET",
+    "requestPath": "http://localhost/api/endpoint"
+  },
+  "rawBody": "raw body received from service",
```

Comment:
We could rename `rawBody` to `responseBody`. It is good that we do not keep it in `response` directly.